### PR TITLE
Travis-ci: Added support ppc64le buffer-more-ints

### PR DESCRIPTION
**Pull request by dthadi3**
I had added ppc64le(Linux on Power) architecture support on Travis-CI in the PR

### CODE REVIEW

File: travis-yml/buffer-more-ints.travis.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# Script License      : MIT License
+#
+# ----------------------------------------------------------------------------
+
+arch:
+  - amd64
+  - ppc64le
+language: node_js
+node_js:
+  - "8"
```
Comment (seth-priya) [inline travis-yml/buffer-more-ints.travis.yml:18]: The node versions seem quite old and out of date, did we try updating these to the latest?

Comment (dthadi3) [reply travis-yml/buffer-more-ints.travis.yml:18]: Yeah, I tried to update latest node_js versions 14 & 15, both amd64 & ppc64le jobs are failing with DeprecationWarning.
https://travis-ci.com/github/dthadi3/node-buffer-more-ints/builds/211269183

File: travis-yml/buffer-more-ints.travis.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : buffer-more-ints
+# Source Repo         : https://github.com/dpw/node-buffer-more-ints
+# Travis Job Link     : https://travis-ci.com/github/dthadi3/node-buffer-more-ints/builds/211017296
+# Created travis.yml  : No
+# Maintainer          : Devendranath Thadi <devendranath.thadi3@gmail.com>
+#
+# Script License      : MIT License
```
Comment (seth-priya) [inline travis-yml/buffer-more-ints.travis.yml:9]: Please update the license to Apache 2.0

**Review by seth-priya — CHANGES_REQUESTED** [review]

File: travis-yml/buffer-more-ints.travis.yml (hunk 2 above)
Comment (dthadi3) [reply travis-yml/buffer-more-ints.travis.yml]: Done

**Review by seth-priya — APPROVED** [review]

**Pull request merged**